backend/src/main/java/gov/cdc/usds/simplereport/api/devicetype/DeviceTypeController.java

```diff
@@ -0,0 +1,22 @@
+package gov.cdc.usds.simplereport.api.devicetype;
+
+import static gov.cdc.usds.simplereport.config.BeanProfiles.PROD;
+
+import gov.cdc.usds.simplereport.service.DeviceTypeSyncService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Profile("!" + PROD)
```

Comment:
We need to remove this to enable it on prod since the @scheduled stuff didnt work